python/core/auto_generated/qgsbrowserproxymodel.sip.in

```diff
@@ -131,6 +131,21 @@ filterByLayerType() is ``True``.
 .. seealso:: :py:func:`layerType`
 
 .. seealso:: :py:func:`setFilterByLayerType`
+%End
+
+    void setDataItemProviderKeyFilter( const QStringList &filter );
+%Docstring
+Sets the customization filters for data items based on item's data provider key
+
+By default browser model shows all items from all available data items provider and few special
+items (e.g. Favourites). To customize the behavious, set the filter to not load certain data items.
+The items that are not based on data item providers (e.g. Favourites, Home) have
+prefix "special:".
+
```

Comment:
I'd suggest adding some specific examples here - eg. the string for postgres and the string for the favourites or home item -- it's not immediately clear from the docs alone.